tests/ops/qubit/test_qubit_observablers.py

```diff
@@ -0,0 +1,376 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
```

Comment:
Can't suggest, but the name of the file needs adjustment: `test_qubit_observablers.py` -> `test_qubit_observables.py`.